src/main/java/com/mycompany/app/App.java

```diff
@@ -12,6 +12,10 @@
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+                System.out.format("No %s",1);
+        if ("1" == "2") {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.